Doc/library/calendar.rst

```diff
@@ -138,6 +138,32 @@ interpreted as prescribed by the ISO 8601 standard.  Year 0 is 1 BC, year -1 is
 
    :class:`TextCalendar` instances have the following methods:
 
+   .. method:: formatday(theday, weekday, width=2)
+
+      Return a string representing a single day formatted with the given width
```

Comment:
Typo? Also, a comma is probably needed here.
```suggestion
      Return a string representing of a single day formatted with the given width,
```

Reply:
I don't believe "of" is necessary and neither is the comma.